src/js/utils/Units.js

```diff
@@ -4,7 +4,7 @@ const Units = {
   formatResource(resource, value) {
     value = Maths.round(value, 2);
 
-    if (resource !== "cpus") {
+    if (resource !== "cpus" && resource !== "gpus") {
```

Comment:
If I'm not mistaken this should be an or `||` instead of and `&&`

Reply:
In that case if the resource is "cpus" and not "gpus" it will run, which we don't want to.

Reply:
ok thanks.